service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/NoticeOfProceedingsControllerAboutToSubmitTest.java

```diff
@@ -104,7 +104,7 @@ private MvcResult makeRequest(CallbackRequest request) throws Exception {
 
     private Map<String, Object> createTemplatePlaceholders() {
         return Map.of(
-            "courtName", "Swansea Family Court",
+            "courtName", "Family Court sitting at Swansea",
```

Comment:
Why'd you change this?

Reply:
We don't use Swansea Family Court anymore, we use the other format for court names - but since this is just testing and the court name change isn't relevant to the story I'll change it back